src/remove.js

```diff
@@ -49,6 +51,28 @@ async function main() {
     : force delete the CRD and CR instances without allowing the controller to clean up children (Default false)
     `);
     return;
+  } else if (argv.d || argv.debug) {
```

Comment:
Seems like there is something missing from our organization if we have the exact same code duplicated.

Reply:
seemed excessive to put just this in its own module